app/shared/commonProperties.js

```diff
@@ -7,6 +7,15 @@ function sameProperty(prop1, prop2) {
   );
 }
 
+function isSelect(property) {
```

Comment:
I understand this is a new function, but the general concept in v1 is that if it inherits a (multi)select, it is a relationship property. I would rename it accordingly (something like `isOrInheritsSelect`) to avoid confusion.